apps/web/src/components/RightSideBarActivities.tsx

```diff
@@ -68,7 +68,7 @@ export const RightSideBarActivites: React.FC = () => {
               type='checkbox'
               name='selectAll'
               id='selectAll'
-              className='mr-3 h-5 w-5 min-w-5'
+              className='mr-3 h-5 w-5 min-w-5 text-bcDarkBlue accent-bcBlueLink'
```

Comment:
The text isn't blue for the checkbox. Refer https://www.figma.com/file/QuGyUBu1JEkNaGTb8ZfC3l/Care-mapping-design?node-id=626%3A10124

Reply:
Sorry, got confused, cuz the text colour for the occupation checkbox labels is #313132 and isn't pure black like here. Reverting.